client/src/stories/6-EntryInput.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import EntryInput from '../components/EntryInput';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Entry Input'
+};
+
+export function JournalEntryInput(props) {
+  return (
+    <div onChange={action('key was pressed')} {...props}>
```

Comment:
This doesn't look correct. Stories won't have props injected.
And the onChange should be on the input and not on the div

Reply:
Maybe here it s not necessary but, for example, the UniversalButton Component will lose hover and active state, when I remove the properties in the story. Is this really intended?